docs/reference/modules/ROOT/pages/building.adoc

```diff
@@ -1,20 +1,88 @@
-= Building Crux artifacts
+= Building
 
-Alongside the various images available on https://hub.docker.com/u/juxt[*Dockerhub*], there are a number of artifacts available for getting started quickly with Crux.
+`crux-build` is a module within the crux repo with two key objectives:
 
-These can be found on the latest https://github.com/juxt/crux/releases[release] of Crux.
-Currently, these consist of a number of common configuration *uberjars* and a custom artifact builder.
+* Providing pre-configured JARs and Docker containers to start a basic implementation of Crux with no prior knowledge required.
+* Providing a mechanism to spin up custom Crux artifacts - with two necessary inputs:
+** A `deps.edn` file containing the dependencies that the node requires.
+** A `crux.edn` file, containing the configuration options to start the node with.
 
-To create your own custom artifacts for Crux:
+== Preconfigured Artifacts
 
-* Download and extract the `crux-builder.tar.gz` from the latest release
-* You can build an uberjar using either Clojure's `deps.edn` or Maven (whichever you're more comfortable with)
-** For Clojure, you can add further Crux dependencies in the `deps.edn` file, set the node config in crux.edn, and run `build-uberjar.sh`
-** For Maven, it's the same, but dependencies go in `pom.xml`
-* Additionally, you can build a Docker image using the `build-docker.sh` script in the **docker** directory.
+=== `crux-standalone`
 
-== Docker
+Available since `20.01-1.7.0-alpha`, on both:
```

Comment:
Don't worry about `available since` in the reference - the user's already looking at a specific version